posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -573,6 +582,55 @@ public class PostHogReplayIntegration(
         }
     }
 
+    private fun findMaskableComposeRects(
+        view: View,
+        rectsToMask: MutableList<Rect>,
+    ) {
+        val semanticsOwner =
+            (view as? RootForTest)?.semanticsOwner ?: run {
+                config.logger.log("View is not a RootForTest: $view")
+                return
+            }
+        val semanticsNodes = semanticsOwner.getAllSemanticsNodes(true)
+
+        semanticsNodes.forEach { node ->
+            val isTextInput = node.config.contains(SemanticsProperties.EditableText)
```

Comment:
I think there are more options here such as `SemanticsProperties.Text`, etc

Reply:
But why do we need `SemanticProperties.Text`. For `maskAllTextInputs`, `SemanticsProperties.EditableText` is enough right?

Reply:
`maskAllTextInputs` is for every type of text, whether it is a text (like just a label or something), an editable text (an edit text), etc.

Reply:
Ah, okay 👍